src/main/java/com/example/LooseCouplingDemo/dto/AddMessagesDTO.java

```diff
@@ -0,0 +1,12 @@
+package com.example.LooseCouplingDemo.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+
+@Data
+public class AddMessagesDTO {
```

Comment:
```suggestion
public class MessageAdditionDTO {
```
The name sounds more like a method name. 
Since this class is used for adding only one message the name should be singular.

Follow these suggestions for the other DTOs too.

Reply:
Done